content/applications/inventory_and_mrp/manufacturing/management/configure_manufacturing_product.rst

```diff
@@ -0,0 +1,73 @@
+===================================
+Manufacturing product configuration
+===================================
+
+.. |BOM| replace:: :abbr:`BoM (Bill of Materials)`
+
+In order to manufacture a product in Odoo *Manufacturing*, the product must be properly configured.
+Doing so consists of enabling the Manufacturing route and configuring a Bill of Materials (BoM) for
+the product. Once these steps are completed, the product is selectable when creating a new
+manufacturing order.
+
+Activate the Manufacture route
+==============================
+
+The Manufacture route is activated for each product on its product page. To do so, begin by
+navigating to :menuselection:`Manufacturing --> Products --> Products`, then select an existing
+product, or create a new one by clicking :guilabel:`New`.
+
+On the product page, select the :guilabel:`Inventory` tab, then enable the :guilabel:`Manufacture`
+checkbox in the :guilabel:`Routes` section. This tells Odoo the product can be manufactured.
+
+.. image:: configure_manufacturing_product/manufacturing-route.png
+   :align: center
+   :alt: The Manufacturing route on the Inventory tab of a product page.
+
+Configure a Bill of Materials (BoM)
```

Comment:
headings are supposed to be in sentence case 
bill of materials is not a proper noun